app/models/aimodels/AiModel.scala

```diff
@@ -36,25 +40,50 @@ class AiModelService @Inject()(dataStoreDAO: DataStoreDAO,
                                dataStoreService: DataStoreService,
                                userDAO: UserDAO,
                                userService: UserService,
+                               organizationDAO: OrganizationDAO,
                                jobDAO: JobDAO,
                                jobService: JobService) {
-  def publicWrites(aiModel: AiModel)(implicit ec: ExecutionContext, ctx: DBAccessContext): Fox[JsObject] =
+  def publicWrites(aiModel: AiModel, requestingUser: User)(implicit ec: ExecutionContext,
+                                                           ctx: DBAccessContext): Fox[JsObject] =
     for {
       dataStore <- dataStoreDAO.findOneByName(aiModel._dataStore)
-      user <- userDAO.findOne(aiModel._user)
-      userJs <- userService.compactWrites(user)
+      user <- userDAO
```

Comment:
maybe `userOpt`? Also, maybe we have a box2Option conversion somewhere that could be used Here? Or if not, maybe create it. Ideally, I would see something like `Fox.runOptional(userBox.toOption)(userService.compactWrites)` but I don’t know if we have everything to make that happen.